### PR TITLE
rebase to 1.9

### PR DESCRIPTION
**Pull request by deads2k**

### CODE REVIEW

File: vendor/k8s.io/apiserver/pkg/admission/plugin/webhook/namespace/matcher.go [hunk 1]
```diff
@@ -93,7 +93,7 @@ func (m *Matcher) MatchNamespaceSelector(h *v1beta1.Webhook, attr admission.Attr
 		// namespace, it is exempted from all webhooks for now.
 		// TODO: figure out a way selective exempt cluster scoped resources.
 		// Also update the comment in types.go
-		return false, nil
```
Comment (sttts) [inline vendor/k8s.io/apiserver/pkg/admission/plugin/webhook/namespace/matcher.go]: 😢

Comment (deads2k) [reply vendor/k8s.io/apiserver/pkg/admission/plugin/webhook/namespace/matcher.go]: Yeah, we're going to have to update the code.  I'm thinking we apply the same label selector.

Comment (sttts) [reply vendor/k8s.io/apiserver/pkg/admission/plugin/webhook/namespace/matcher.go]: Compare https://github.com/kubernetes/kubernetes/issues/57964, started discussion with @caesarxuchao there.

**Pull request merged**